### PR TITLE
Fix several issues with oneof responses

### PR DESCRIPTION
**Pull request by ezilber-akamai**
## 📝 Description

Fixed `oneOf` response properties being overwritten during schema aggregation. When a `oneOf` defines the same property across multiple branches, keep the most complete definition instead of letting a later, emptier branch overwrite it. This fixes fields like `public.ipv6.ranges` being dropped from output.

Also fixed an issue with oneOf schemas in `_parse_response_model` that was causing nested response data to be displayed incorrectly.

## ✔️ How to Test

`make test-unit`
`make test-int TEST_CASE=test_interface_update` (was previously failing)
`make test-int TEST_SUITE=firewalls` (was previously failing)

### CODE REVIEW

**Comment by mawilk90** [comment]
The tests passed locally. Thanks for addressing it!

**Review by mawilk90 — APPROVED** [review]

File: linodecli/baked/util.py [hunk 1]
```diff
@@ -8,6 +8,63 @@
...
+    # richness of each branch.
+    for composition_field in ("oneOf", "allOf", "anyOf"):
+        for branch in get(schema, composition_field) or []:
+            score += 1 + _schema_richness(branch, _depth + 1)
+
+    # Account for array item schemas so arrays of objects are scored by their
+    # element structure.
+    array_items = get(schema, "items")
+    if array_items is not None:
+        score += _schema_richness(array_items, _depth + 1)
```
[the remark this answers is not in the document]
Comment (yec-akamai) [reply linodecli/baked/util.py:64]: I think we can apply this comment for the edge case

**Review by yec-akamai — APPROVED** [review]
Tests passed locally!
